[BUGFIX] Fix docs_integration pipeline

kenwade4 opened the pull request:
Changes proposed in this pull request:
- Install aws-glue-libs from github repo in docusaurus_tests stage
- Use SparkContext.getOrCreate() in aws_glue_deployment_patterns.py
- Use correct suite name in bigquery integration test
- Use the name of the temp table instead of 'ge_temp' in bigquery integation test
- Use correct validator var in bigquery integration test
- Add requirements-dev-azure.txt to requirements-dev.txt
- Ignore type checking on 4 variables
- Add some logger.debug messages

kenwade4 committed:
Install aws-glue-libs from github repo in docusaurus_tests stage

kenwade4 committed:
Add -s flag to pytest in docusaurus_tests (to see more potentially useful output)

kenwade4 committed:
Add some logger.debug messages

ghost commented:
<details>
<summary><b>👇 <a href="https://app.codesee.io/r/reviews?pr=5734&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations">Click on the image</a> for a new way to code review</b>
</summary>

  - Make big changes easier — review code in small groups of related files
  - Know where to start — see the whole change at a glance
  - Take a code tour — explore the change with an interactive tour
  - Make comments and review — all fully sync’ed with github

    [Try it now!](https://app.codesee.io/r/reviews?pr=5734&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations)
</details>

[![Review these changes using an interactive CodeSee Map](https://s3.us-east-2.amazonaws.com/maps.codesee.io/images/github/great-expectations/great_expectations/5734/ef31b923/24ecacb9888fb38f2e8d757eccc95d0244c7cdec.svg)](https://app.codesee.io/r/reviews?pr=5734&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations)

#### Legend
![CodeSee Map Legend](https://app.codesee.io/Legend_v2.png)

kenwade4 committed:
Use SparkContext.getOrCreate() in aws_glue_deployment_patterns.py

Causing ValueError: Cannot run multiple SparkContexts at once

See https://stackoverflow.com/a/47731444

kenwade4 committed:
Use correct suite name in bigquery integration test

great_expectations.exceptions.exceptions.DataContextError: expectation_suite test_suite_2 not found

kenwade4 committed:
Add debug messages in ConfiguredAssetAzureDataConnector when BlobServiceClient fails

kenwade4 committed:
Reformat with black

kenwade4 committed:
Ignore type checking on 4 variables

kenwade4 committed:
Temporarily just run those that are failing in test_docs job

kenwade4 committed:
Merge remote-tracking branch 'origin/develop' into bug/docs_integration_tests

kenwade4 committed:
Don't include -s for more pytest output in docs-integration

kenwade4 committed:
Show some env var names in docs-integration and find jars directories

kenwade4 committed:
Merge remote-tracking branch 'origin/develop' into bug/docs_integration_tests

kenwade4 committed:
Only search in /opt for jars and get some java info

kenwade4 committed:
Use the name of the temp table instead of 'ge_temp' in bigquery integration test

kenwade4 committed:
Use correct validator var in bigquery integration test

Also, "the truth value of a DataFrame is ambiguous" so don't just assert
the dataframe that comes back from validator.head(..)

kenwade4 committed:
Add requirements-dev-azure.txt to requirements-dev.txt

kenwade4 committed:
Remove the temp debugging/troubleshooting stuff that shouldn't be merged

kenwade4 committed:
Merge remote-tracking branch 'origin/develop' into bug/docs_integration_tests

Kilo59 commented on requirements-dev.txt:
```diff
@@ -4,6 +4,7 @@
 --requirement requirements-dev-sqlalchemy.txt
 
 --requirement requirements-dev-arrow.txt
+--requirement requirements-dev-azure.txt
```
Does this need to be added to these lists?
https://github.com/great-expectations/great_expectations/blob/develop/great_expectations/core/usage_statistics/package_dependencies.py#L67-L151

Or are all its contents already included?

kenwade4 replied:
It's already included.

Shinnnyshinshin reviewed (approved):
Can't thank you enough for this @kenwade4. LGTM pending @Kilo59 's question

kenwade4 committed:
Update the 'install dependencies' step of docs_integration pipeline

kenwade4 committed:
Merge remote-tracking branch 'origin/develop' into bug/docs_integration_tests

kenwade4 committed:
Merge branch 'develop' into bug/docs_integration_tests
